app.js

```diff
@@ -0,0 +1,97 @@
+
+const myMap = {
+	currentLocation: [],
+	businesses: [],
+	map: {},
+	markers: {},
+	buildMap() {
+		this.map = L.map('map', {
+		center: this.currentLocation,
+		zoom: 11,
```

Comment:
small nit: Try indenting things such that each level of indentation is it's own scope